core/src/main/scala/org/genivi/sota/core/transfer/VehicleUpdates.scala

```diff
@@ -113,8 +158,10 @@ object VehicleUpdates {
       .result
       .headOption
       .flatMap {
-        case Some(((ns: Namespace, uuid: UUID, updateVin: Vehicle.Vin, status: UpdateStatus.UpdateStatus),
-                   updateRequest: UpdateRequest)) =>
+        case Some((
+          (ns: Namespace, uuid: UUID, updateVin: Vehicle.Vin, status: UpdateStatus.UpdateStatus, installPos: Int),
```

Comment:
That list has grown over time (namespace added, installPos added).
Without the type annotations will more readable, agreed.